.github/workflows/firebase_auth.yaml

```diff
@@ -27,9 +27,13 @@ jobs:
       - name: "Install Flutter"
         run: ./.github/workflows/scripts/install-flutter.sh stable
       - name: "Install Tools"
-        run: ./.github/workflows/scripts/install-tools.sh
+        run: |
+          ./.github/workflows/scripts/install-tools.sh
+          sudo npm i -g firebase-tools
```

Comment:
Definitely! I have a separate pr going where I'll de flake ci and add lint. Didn't want to change style here though so reviews stayed focused. This will flake though as is